README.md

```diff
@@ -0,0 +1,79 @@
+# Drone Fork Approval Extension
+
+:electric_plug: Fork Approval is a simple [Drone Validation Extension](https://docs.drone.io/extensions/validation/)
+that ensures every PR that comes from a fork must have its Drone CI build
+approved before it will run.
+
+Perhaps you're worried about OSS contributors running crypto miners?
+Perhaps you'd like to use privileged containers in your PR pipeline?
+
+Fork Approval can help you do so safely!
+
+## Usage
+
+1. Create a shared secret:
+
+```console
+$ DRONE_SECRET="$(openssl rand -base64 32)"
+```
+
+2. Run the container:
+
+```console
+$ docker run --detach \
+  --restart=always \
+  --publish=3080:80 \
```

Comment:
nit: `3080` is Teleport's default web UI port; might be worth making this `3000` or `8080` just to avoid the risk of any possible confusion.

Reply:
Changed in 6f2087b to 3888.